File: .sdk/tm/php/utility/MakeContext.php
<?php
declare(strict_types=1);

// HubspotCommunicationPreferences SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class HubspotCommunicationPreferencesMakeContext
{
    public static function call(array $ctxmap, ?HubspotCommunicationPreferencesContext $basectx): HubspotCommunicationPreferencesContext
    {
        return new HubspotCommunicationPreferencesContext($ctxmap, $basectx);
    }
}

File: .sdk/tm/py/pkg/utility/make_context.py
# HubspotCommunicationPreferences SDK utility: make_context

from projectname_sdk.core.context import HubspotCommunicationPreferencesContext


def make_context_util(ctxmap, basectx):
    return HubspotCommunicationPreferencesContext(ctxmap, basectx)
